fish_prompt.fish

```diff
@@ -92,8 +92,19 @@ function __bobthefish_pretty_parent -S -a current_dir -d 'Print a parent directo
   string replace -ar '(\.?[^/]{'"$fish_prompt_pwd_dir_length"'})[^/]*/' '$1/' "$parent_dir/"
 end
 
+function __ignore_vcs_dir -d 'Identifies if dir is to be ignored for VCS highlighting'
+  for path_ in $vcs_ignore_paths
+    if [ (string match -r (realpath $path_ ^/dev/null) (realpath $PWD ^/dev/null)) ]
```

Comment:
this probably should be a regular match (i.e. glob) rather than a regex match.

Reply:
doh, of course you are right... I'll fix it..